README.md

```diff
@@ -33,12 +33,13 @@ the function `getStatus` is passed the current time and an optional options obje
 Currently the only valid option is a boolean field - `next`. When
 `next` set to a truthy value the returned object will contain 2 additional
 fields, `nextClosed` and `nextOpen`. Both fields are a
-[moment with a time zone](https://momentjs.com/timezone/docs/#/using-timezones/parsing-in-zone/).
+[moment with a time zone](https://momentjs.com/timezone/docs/#/using-timezones/parsing-in-zone/)
+indicating the next time the place is closed and open, respectively.
 
 ### Example usage
 
-Given the open hours of a place is represented by the following set of opening times:
-Note each day can have a maximum of 3 open sessions.
+Given the open hours of a place is represented by the following set of opening times
+(note a maximum of 3 open sessions per day is supported):
```

Comment:
I don't believe this is true. I think the number of sessions are unlimited. Overlapping of sessions is not allowed but there are probably tests which should be added to clarify behaviour around this.